build.gradle

```diff
@@ -18,6 +18,14 @@ buildscript {
   }
 }
 
+allprojects {
+  gradle.projectsEvaluated {
```

Comment:
I thought I would include it because the change is so small. I see your point though, I'll back it out.